src/helpers/collateralsApy/index.js

```diff
@@ -47,16 +47,19 @@ export const fetchTokenApy = async (pool) => {
     if (pool.config.id === 31 || pool.config.id === 32)
       return await getStargateApy(pool, provider);
 
-    if (pool.config.id === 39) return await getApeApy(provider);
+    if (pool.config.id === 39) return await getMegicApeApy(provider);
   }
 
   if (chainId === 10) {
     if (pool.config.id === 1) return await getVeloApy(pool, provider);
   }
 
   if (chainId === 42161) {
-    if (pool.config.id === 2 || pool.config.id === 3)
-      return await getGlpApy(pool.config.id === 3);
+    if (pool.config.id === 2 || pool.config.id === 3) {
+      const response = await getMegicGlpApy(chainId);
```

Comment:
suggestion: use pool.config.apyType enum with switch case

Reply:
This file is not related to the Stake refactoring. I'll fix it in the next updates